### PR TITLE
feat: connector i18n

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Change connector's metadata format to support i18n.

By the way, do we have standard language key (en, zh_CN ...) enums? @gao-sun

### CODE REVIEW

File: packages/core/src/connectors/types.ts [hunk 1]
```diff
@@ -3,9 +3,9 @@ import { ConnectorType } from '@logto/schemas';
 export interface ConnectorMetadata {
   id: string;
   type: ConnectorType;
-  name: string;
+  name: Record<string, string>;
```
Comment (simeng-li) [inline packages/core/src/connectors/types.ts:6]: Record <enum, string> ?

**Review by gao-sun — COMMENTED** [review]
The key standard should follow ISO / `i18next`. See `i18next` docs for details.

File: packages/core/src/connectors/types.ts (hunk 1 above)
Comment (wangsijie) [reply packages/core/src/connectors/types.ts:6]: We currently don't have a language code enums in this project, keep the same for now?

Comment (simeng-li) [reply packages/core/src/connectors/types.ts:6]: k

Comment (gao-sun) [reply packages/core/src/connectors/types.ts:6]: actually `keyof resource` do the trick. would you like to add an export like `type Languages = keyof resource;` in `packages/phrases/src/index.ts`?

**Review by gao-sun — CHANGES_REQUESTED** [review]

**Review by simeng-li — APPROVED** [review]
LGTM, unblock the merge process once all conversations are resolved.

**Review by gao-sun — APPROVED** [review]
lgtm

**Pull request merged**